eng/targets/RepoToolset/ProjectDefaults.props

```diff
@@ -90,50 +88,22 @@
     </When>
   </Choose>
 
-  <Import Project="|Build precondition failed: Unable to determine the language of the project. Consider adding Directory.Build.props that sets Language property to directory '$(MSBuildProjectDirectory)'.|"
-          Condition="'$(Language)' == '' and '$(MSBuildProjectExtension)' != '.shproj' and '$(MSBuildProjectExtension)' != '.swixproj' and '$(MSBuildProjectExtension)' != '.vsmanproj'"/>
-
   <Choose>
     <!-- C# specific settings -->
     <When Condition="'$(Language)' == 'C#'">
       <PropertyGroup>
-        <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
         <NoWarn>$(NoWarn);1701;1702;1705;1591</NoWarn>
       </PropertyGroup>
-
-      <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
-        <DebugSymbols>true</DebugSymbols>
-        <DefineConstants>DEBUG;TRACE</DefineConstants>
-      </PropertyGroup>
-
-      <PropertyGroup Condition="'$(Configuration)' == 'Release'">
-        <DefineConstants>TRACE</DefineConstants>
-        <Optimize>true</Optimize>
-      </PropertyGroup>
     </When>
 
     <!-- VB specific settings -->
     <When Condition="'$(Language)' == 'VB'">
       <PropertyGroup>
         <MyType>Empty</MyType>
         <OptionCompare>Binary</OptionCompare>
-        <OptionExplicit>On</OptionExplicit>
-        <OptionInfer>On</OptionInfer>
         <OptionStrict>On</OptionStrict>
-        <VBRuntime>Embed</VBRuntime>
         <RemoveIntegerChecks>true</RemoveIntegerChecks>
```

Comment:
I would expect the namespace clearing to be here

Reply:
Namespace clearing was a no-op.